Mobile.BuildTools/build/Mobile.BuildTools.props

```diff
@@ -16,6 +16,7 @@
     <IsMacOSProject>false</IsMacOSProject>
     <IsTizenProject>false</IsTizenProject>
     <MobileBuildToolsTargets>$(MSBuildThisFileDirectory)Mobile.BuildTools.targets</MobileBuildToolsTargets>
+    <DisableBuildToolsArtifactCopy>false</DisableBuildToolsArtifactCopy>
```

Comment:
This does not need to be there. That is why the conditions explicitly check value rather than treating it as a boolean.

Reply:
ok, can I add this to readme in tokenize section?
Because there no mentions about it in documentation

Reply:
yes please do.